docs/processes/cldr-icu.md

```diff
@@ -472,12 +472,15 @@ mvn install -pl main/common_tests -Dtest=MeasureUnitTest#TestGreek
 13b. Optionally run the tests in exhaustive mode
 
 Optionally run before committing changes, or run to diagnose failures from
-running exhaustive CI tests in the PR using `/azp run CI-Exhaustive`:
+running exhaustive CI tests in the PR using exhaustive localy:
 ```sh
 cd $ICU4J_ROOT
 mvn install -DICU.exhaustive=10 2>&1 | tee $NOTES/icu4j-newData-mvnCheckEx.txt
 ```
 
+Exhaustive tests in CL can ve triggered by running the "Exhaustive Tests for ICU"
+action from the GitHub web UI.
```

Comment:
typo fix & adding link:
```
Exhaustive tests in CL can be triggered by running the "Exhaustive Tests for ICU"
action from the GitHub web UI:
https://unicode-org.github.io/icu/userguide/dev/ci.html#exhaustive-tests
```
except nicer if we can link to the .md page with a relative link, and GitHub Pages turns that into the link to the .html page

Reply:
Added a link, hope it works.
I used CI (not CL) intentionally. Because that is a CI test.

Reply:
I know, I had missed the "CL" typo